### PR TITLE
Updated content on 'installing scientific packages' page

### LINKED ISSUE #383: Scientific packages guide out of date
The guide on [Installing Scientific Packages](https://packaging.python.org/guides/installing-scientific-packages/) talks about how NumPy and SciPy can't be distributed as wheels; that seems to have been [resolved](https://pypi.org/project/numpy/#files) since this page was updated (2014?).

### PR DESCRIPTION
**Pull request by waseem18**
Fixes #383

### CODE REVIEW

**Comment by waseem18** [comment]
@jonparrott Can you please review this?

Thanks

File: source/guides/installing-scientific-packages.rst [hunk 1]
```diff
@@ -137,21 +130,21 @@ be loaded and unloaded from the user's environment.
...
-for native virtual environments. Conda makes environments first-class citizens, 
-making it easy to create independent environments even for C libraries. It is 
-written in Python, but is Python-agnostic. Conda manages Python itself as a 
-package, so that `conda update python` is possible, in contrast to pip, which only 
-manages Python packages. Conda is available in Anaconda and Miniconda 
-(an easy-to-install download with just Python and conda). 
+packages, it is not limited to just Python packages. It has full support for
+native virtual environments. Conda makes environments first-class citizens,
+making it easy to create independent environments even for C libraries. It is
+written in Python, but is Python-agnostic. Conda manages python itself as a
```
Comment (brainwane) [inline source/guides/installing-scientific-packages.rst]: Quick question, why did you lower-case "python" here?

Comment (waseem18) [reply source/guides/installing-scientific-packages.rst]: I missed it. Will Capitalize it

Comment (waseem18) [reply source/guides/installing-scientific-packages.rst]: Addressed

**Comment by waseem18** [comment]
@brainwane I missed changing `Last Reviewed` - changing it to today's date. What about `Page Status`? Currently it says `Incomplete`.

**Comment by ncoghlan** [comment]
I'm thinking those "Last Reviewed" and "Page Status" markers may have served their purpose now, and we'd be best off dropping them entirely.

IIRC, we added them back around 2014 when our tooling recommendations were sometimes changing from month to month as we discovered new problems with the various available options, and the PyPUG wasn't well enough known yet for corrections to be sent in as PRs.

**Comment by waseem18** [comment]
Thanks for the response @ncoghlan . So shall I go ahead and remove `Last Reviewed` and `Page Status` ?

**Comment by ncoghlan** [comment]
@waseem18 Aye, go ahead.

**Comment by brainwane** [comment]
@waseem18 ready to be re-reviewed?

**Comment by waseem18** [comment]
Yes @brainwane - I've removed the `Last Reviewed` and `Page Status` as per @ncoghlan 's review.

File: source/guides/installing-scientific-packages.rst [hunk 2]
```diff
@@ -22,23 +19,16 @@ for most of the software in the `scientific Python stack
...
-time.
-
-It is expected that this situation will eventually be resolved either by
-future iterations of the standard tools providing full support for the
-intricacies of NumPy's current build and distribution process, or by the
-NumPy developers choosing one build variant as the "lowest acceptable
-common denominator" and publishing that as a wheel file on PyPI.
+Starting with version 1.10.4 of NumPy and version 1.0.0 of SciPy, pre-built
+32-bit and 64-bit binaries in the ``wheel`` format are available for all major
+operating systems (Windows, macOS, and Linux) on PyPI. Note however that on
```
Comment (brainwane) [inline source/guides/installing-scientific-packages.rst]: should be:

> Note, however, that

Comment (waseem18) [reply source/guides/installing-scientific-packages.rst]: Addressed.

**Pull request merged**